server/bootstrap/src/main/resources/logback.xml

```diff
@@ -15,12 +15,6 @@
   limitations under the License.
 -->
 <configuration>
-    <appender name="console" class="ch.qos.logback.core.ConsoleAppender">
-        <encoder>
-            <pattern>%d{ISO8601} [%thread] %-5level %logger{36} - %msg%n</pattern>
-        </encoder>
-    </appender>
-
     <appender name="fileLogAppender" class="ch.qos.logback.core.rolling.RollingFileAppender">
         <file>${server_log_dir}/kaa-bootstrap-server${server_log_sufix}.log</file>
         <rollingPolicy class="ch.qos.logback.core.rolling.TimeBasedRollingPolicy">
```

Comment:
Please ensure that file rotation is working and add smth that trigger log rotation by size. Let's keep logs with max 100Mb size.
```
<timeBasedFileNamingAndTriggeringPolicy class="ch.qos.logback.core.rolling.SizeAndTimeBasedFNATP">
    <maxFileSize>100MB</maxFileSize>
</timeBasedFileNamingAndTriggeringPolicy>
```